sdk/appconfiguration/azure-appconfiguration/samples/README.md

```diff
@@ -0,0 +1,40 @@
+---
+page_type: sample
+languages:
+  - python
+products:
+  - azure
+  - azure-app-configuration
+---
+
+# Azure App Configuration Data Library Python Samples
+
+## Prerequisites
+
+You must have an [Azure subscription][azure_sub], and a [Configuration Store][configuration_store] to use this package.
+
+To create a Configuration Store, you can use the Azure Portal or [Azure CLI][azure_cli].
+
+After that, create the Configuration Store:
```

Comment:
I would combine this with the previous line to make sure people do not think there is a next step:

To create a Configuration Store, you can either use Azure Portal[link to portal] or if you are using [Azure CLI][azure_cli] you can simply run the following snippet in your console: 
`that cli line you had there`